MyTasks/Task2_2/СurrencyСonverter.cs

```diff
@@ -0,0 +1,40 @@
+﻿
+Console.WriteLine("Здравствуйте! Введите сумму, которую хотите конвертировать. Например: 1000");
+double moneyAmount;
+moneyAmount = double.Parse(Console.ReadLine());
+
+Console.WriteLine("Выберите валюту конвертации: USD, EUR, RUB, CAD");
+var currencyName = Console.ReadLine();
+double sum;
+if (currencyName == "USD")
+{
+    sum = moneyAmount / 2.5712;
+    var round = Math.Round(sum, 2, MidpointRounding.ToEven);
+    var result = round - (moneyAmount * 0.03);
+    Console.WriteLine(result + " " + currencyName);
+}
+else if (currencyName == "EUR")
+{
+    sum = moneyAmount / 2.9314;
+    var round = Math.Round(sum, 2, MidpointRounding.ToEven);
+    var result = round - (moneyAmount * 0.03);
+    Console.WriteLine(result + " " + currencyName);
+}
+else if (currencyName == "RUB")
+{
+    sum = moneyAmount / 0.034155;
+    var round = Math.Round(sum, 2, MidpointRounding.ToEven);
+    var result = round - (moneyAmount * 0.03);
+    Console.WriteLine(result + " " + currencyName);
+}
+else if (currencyName == "CAD")
+{
+    sum = moneyAmount / 2.0256;
+    var round = Math.Round(sum, 2, MidpointRounding.ToEven);
+    var result = round - (moneyAmount * 0.03);
+    Console.WriteLine(result + " " + currencyName);
+}
+else
+{
+    Console.WriteLine("Неизвестная валюта.");
+}
```

Comment:
Very bad practise to use million if else. Don't forget about other constructions(For example, switch case)
Create enum class for currency. Storage exchange ratios in Data Class with constants. Where we can get it easily.

Reply:
Is it okay to use public static readonly double[] instead of constants in Data Class? That's how I implemented it in the new "release" :)
As to the rest, used switch-case and created enum class for currency.

Reply:
No, it's not ok. As I said, it is better to use constants to avoid access by indexes in your case, right now you have array of ratios, it's less understandable than it would be with constants

Reply:
@DaryaNikitina, changed to private constants.